DTAdmin/HTTPService.swift

```diff
@@ -0,0 +1,297 @@
+//
+//  HTTPService.swift
+//  DTAdmin
+//
+//  Created by Admin on 18.10.17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import Foundation
+
+class HTTPService {
+    //    static func login (userName:String, password:String){
+    static func login (completion: @escaping (HTTPURLResponse) -> ()){
+        let params = ["username": "admin", "password": "dtapi_admin"]
+        let url = URL(string: "http://vps9615.hyperhost.name/login/index")!
+        var request = URLRequest(url: url)
+        request.httpMethod = "POST"
+        request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        request.setValue("UTF-8", forHTTPHeaderField: "Charset")
+        let jsonData:Data
+        do {
+            jsonData = try JSONSerialization.data(withJSONObject: params, options: [])
+            request.httpBody = jsonData
+        } catch  {
+            print("some error")
+            return
+        }
+        let session = URLSession.shared
+        let task = session.dataTask(with: request) { data, response, error in
+            guard let data = data, error == nil else {
+                print("error=\(String(describing: error))")
+                return
+            }
+            
+            guard let httpStatus = response as? HTTPURLResponse else {
+                return
+            }
+            if httpStatus.statusCode == 200{
+                completion(httpStatus)
+            } else {
+                print("statusCode should be 200, but is \(httpStatus.statusCode)")
```

Comment:
This error is neither meaningful nor user-friendly. Please change it to something meaningful and specific.